admin-client/src/main/java/com/epam/admin_client/AdminClientApplication.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.admin_client;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+
+
+@SpringBootApplication
+public class AdminClientApplication {
+
+	public static void main(String[] args) {
```

Comment:
Please use tai codestyle. This piece of code uses tabs while we agreed on spaces